pom.xml

```diff
@@ -68,7 +68,7 @@
         <version>${dependency.cpf.revision}</version>
       </dependency>
       <dependency>
-        <groupId>pentaho</groupId>
+        <groupId>org.pentaho.ctools</groupId>
         <artifactId>ccc</artifactId>
         <version>${dependency.ccc.revision}</version>
```

Comment:
You could also update the property names, here they still use the deprecated syntax *dependency.#.revision*, but should use *#.version*.
If I remember correctly, dependencies that are ours should use the property names contained in:
https://github.com/pentaho/build-resources/blob/8.1.0.0/version.properties
If for now you don't update CDF at least make sure this rule is enforced in CCC.